internal/cmd/utils.go

```diff
@@ -11,3 +16,16 @@ func LoadContractFromArgs(args []string) (*contract.Contract, error) {
 	}
 	return contract.NewContractFromFile(location)
 }
+
+func MakeGetRequest(uri string) (*http.Response, error) {
+	req, err := http.NewRequestWithContext(context.Background(), http.MethodGet, uri, nil)
+	if err != nil {
+		return nil, err
+	}
+
+	res, err := http.DefaultClient.Do(req)
+	if err != nil {
+		return nil, err
+	}
+	return res, nil
```

Comment:
How about this return http.DefaultClient.Do(req) ??

As caller will check the nil & error case before proceeding ?

Reply:
Yep that makes sense.